api/envoy/extensions/filters/http/router/v3/router.proto

```diff
@@ -106,4 +113,7 @@ message Router {
   // upstream filters will count as a final response if hedging is configured.
   // [#extension-category: envoy.filters.http.upstream]
   repeated network.http_connection_manager.v3.HttpFilter upstream_http_filters = 8;
+
+  // Additional upstream access log options.
+  UpstreamAccessLogOptions upstream_log_options = 9;
```

Comment:
nit: move this next to `upstream_log` above.

Reply:
done